packages/account-v2/src/components/InputWithButton/InputWithButton.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { ComponentProps, forwardRef } from 'react';
+import { Button } from '@deriv-com/ui';
+import { FormInputField } from '../FormFields';
+
+type InputWithButtonProps = ComponentProps<typeof FormInputField> & {
+    buttonText: string;
+    isDisabled?: boolean;
+    isLoading?: boolean;
+    onClick?: () => void;
+};
+
+export const InputWithButton = forwardRef<HTMLInputElement, InputWithButtonProps>(
```

Comment:
may I ask you, why do we need here forwardRef? we even don't use ref here

Reply:
@yauheni-deriv, I believe reusable component have to have the `ref` prop, because some one might want to modify them through a `ref` they pass in. This is especially true, since it might be moved to `@deriv-com/ui`

Reply:
alright, but we don't utilise here ref inside component, right?
so if someone wants to use ref they can modify it later, anyway ref can be passed as a usual props, just with other name.
you can ignore my comments